config/environments/test.rb

```diff
@@ -133,17 +131,6 @@
       Bullet.add_safelist class_name: "CMS::File", type: :n_plus_one_query, association: :provider
       Bullet.add_safelist class_name: "CMS::Page", type: :n_plus_one_query, association: :provider
       Bullet.add_safelist class_name: "CMS::Page", type: :n_plus_one_query, association: :section
-      Bullet.add_safelist class_name: "Cinstance", type: :n_plus_one_query, association: :plan
-      Bullet.add_safelist class_name: "Cinstance", type: :n_plus_one_query, association: :service
-      Bullet.add_safelist class_name: "Cinstance", type: :n_plus_one_query, association: :user_account
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :plan
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :service
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :service
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :service
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :plan
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :user_account
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :user_account
-      Bullet.add_safelist class_name: "Cinstance", type: :unused_eager_loading, association: :service
```

Comment:
I didn't know about this list. Nice to remove items from it

Reply:
https://issues.redhat.com/browse/THREESCALE-9973